pkgs/games/boohu/default.nix

```diff
@@ -0,0 +1,38 @@
+{stdenv, lib, fetchurl, buildGoPackage, ansiTag ? false}:
```

Comment:
What does `ansi` do? And why do we need both versions in nixpkgs?

Reply:
The basic version uses curses-like library [termbox-go](https://github.com/nsf/termbox-go). 
Ansi version does not need external dependencies. It is more portable and should work on all POSIX systems using `stty`.
This is similar to `nethack` and `nethack-x11`. Normal users don't have to deal with complicated things to build a package with this option. They'll just be able to choose `boohu-ansi` instead.
Perhaps I should remove  `platforms`? The package seems to be very portable.

Reply:
We can assume that `cgo` is present on all platforms we support. After all nix is also written in c++. That's why I would only include the version using [tcell](https://github.com/gdamore/tcell).

Reply:
If we can remove `platforms` depends whether `buildGoPackage` sets a default value. Otherwise it will be not build on hydra.

Reply:
I am still not convinced we need both versions.

Reply:
Please decide on one. This is not like nethack where one has the choice between a headless version and a x11 version.

Reply:
I agree with you. PR [here](https://github.com/NixOS/nixpkgs/pull/48138).